src/main/java/net/snowflake/ingest/streaming/internal/InternalStage.java

```diff
@@ -188,6 +188,7 @@ private Optional<String> putRemote(String fullFilePath, byte[] data, int retryCo
         SnowflakeFileTransferAgent.uploadWithoutConnection(
             SnowflakeFileTransferConfig.Builder.newInstance()
                 .setSnowflakeFileTransferMetadata(fileTransferMetadataCopy)
+                .setSilentException(true)
```

Comment:
FWIW, this function name is confusing....

Reply:
cc @sfc-gh-dprzybysz